messages/core_write_access.md

```diff
@@ -0,0 +1,14 @@
+Hello {NAME},
+
+{some information about why they are getting access now}
+
+You can see the formal description of the subpackage maintainers role at http://www.astropy.org/team.html.  But there are some other guidelines to keep in mind, which I'm describing here.  In general you should only merge PRs that are in the domain of your subpackage(s).  This is a guideline rather than a firm rule: if you see something that's straightforward and obvious (e.g., a minor typo fix that's already been approved and the tests have passed but no one hit the merge button), you should feel free to help with that sort of thing even in other packages. Similarly, in areas where there's conflict/uncertainty, it's best to defer to a more senior maintainer (if there is one) or if absolutely necessary the Coordination Committee... but in cases where there's no obvious reason to question it (particularly if the lead is the PR author), it is also fine to merge things when it seems reasonable.
+
+Relatedly, if you haven't already, you should familiarize yourself with the maintainer guidelines: https://docs.astropy.org/en/latest/development/workflow/maintainer_workflow.html.  Please respond indicating you've seen these so we're sure you're aware of them.
```

Comment:
the maintainer guidelines are actually extremely outdated and occasionally factually wrong. There is no way to push to master now, and some of the manual merges are also abandoned around ~v1.0. E.g. this section should be deleted altogether, rather than encouraged to be followed.
https://docs.astropy.org/en/latest/development/workflow/maintainer_workflow.html#integrating-changes-manually

we also don't have backport labels, etc.

Reply:
Hmm.  I see your point, I hadn't actually read it in a while (perhaps ironically...).  Do you think it's better *not* to have maintainer guidelines then, and instead just link them to the basic workflow, or do you think having the maintainer guidelines is a good idea we just need to fix them?

(Either way you're right we should remove/replace with the basic dev workflow *for now* but the question is really about what to do in the astropy repo longer-term)